### PR TITLE
Remove dark/light movie; update contributing page

### PR DESCRIPTION
**Pull request by aeslaughter**

### CODE REVIEW

File: modules/doc/content/framework_development/contributing.md [hunk 1]
```diff
@@ -98,6 +98,16 @@ In addition: [our own slides](https://mooseframework.org/static/media/uploads/do
 
 The main thing to remember when issuing a PR for MOOSE is that all PRs should be specified to go to the `next` branch.
 
+## Pull Request Assignee
+
+In most cases a PR will be assigned to MOOSE developer. The assignee is designated to help ensure
+that the request is reviewed and merged (or canceled), they are not necessarily the reviewer. The aim
+is to make sure every PR is being addressed and monitored. They should also serve as a point of
```
Comment (loganharbour) [inline modules/doc/content/framework_development/contributing.md:105]: 👍🏼

**Review by loganharbour — DISMISSED** [review]

**Comment by dschwen** [comment]
Not autoplaying either. You could try adding a `.play()` jcall in the javascript.Or adding teh `muted` attribute:

https://stackoverflow.com/questions/17994666/video-auto-play-is-not-working-in-safari-and-chrome-desktop-browser

File: framework/doc/content/js/init.js [hunk 2]
```diff
@@ -14,12 +14,4 @@
             alignment: 'left' // Displays dropdown with edge aligned to the left of button
         });
     }); // end of document ready
-
-    // Change video source for dark mode, see media.py RenderVideo
-    if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
-        $('.moose-video source[data-dark-src]').each(function(){
-            var src = $(this).data('dark-src');
-            $(this).attr('src', src);
```
Comment (dschwen) [inline framework/doc/content/js/init.js:22]: You could try 

```
$(this).attr('src', src).play();
```

as the only change to the video stuff.

File: modules/doc/content/framework_development/contributing.md [hunk 3]
```diff
@@ -98,6 +98,16 @@ In addition: [our own slides](https://mooseframework.org/static/media/uploads/do
 
 The main thing to remember when issuing a PR for MOOSE is that all PRs should be specified to go to the `next` branch.
 
+## Pull Request Assignee
+
+In most cases a PR will be assigned to MOOSE developer. The assignee is designated to help ensure
+that the request is reviewed and merged (or canceled), they are not necessarily the reviewer. The aim
```
Comment (joshuahansel) [inline modules/doc/content/framework_development/contributing.md:104]: run-on. One solution is to replace "," with ";"

**Comment by aeslaughter** [comment]
This version worked on all three browsers for me: Firefox, Chrome, Safari. I am done. If anyone else want to improve please do.

**Review by loganharbour — APPROVED** [review]

**Pull request merged**